.Jules/knowledge.md

```
@@ -211,6 +240,20 @@ interface BalanceSummary {
 
 ---
 
+## Testing & Verification
+
+### Playwright Verification Patterns
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)